os/drivers/lwnl/Make.defs

```diff
@@ -18,7 +18,12 @@
 
 # Include lwnl drivers
 ifeq ($(CONFIG_LWNL80211),y)
-	CSRCS += lwnl.c lwnl_evt_queue.c lwnl_ioctl.c lwnl_netdev.c
+CSRCS += lwnl.c lwnl_evt_queue.c
+
+ifeq ($(CONFIG_NET_NETMGR), y)
```

Comment:
If you want to add `lwnl_ioctl.c` and `lwnl_netdev.c` only when `CONFIG_NET_NETMGR` is not enabled,
how about using `ifneq`?